index.bs

```diff
@@ -1298,9 +1299,9 @@ Gets the {{Error}} object passed to {{SensorErrorEventInit}}.
         is a [=nested browsing context=] whose [=active document=]'s [=origin=]
         is not [=same origin-domain=] as |document|'s [=origin=],
         then return "insecure".
-    1.  If the [=currently focused area=] is in a different [=top-level browsing context=]
-        than the current [=top-level browsing context=],
-        then return "insecure".
+    1.  Let |has_focus| be the result of running the [=has focus steps=]
+        passing it |document| as argument.
```

Comment:
But it's: "passing it |foo| as arg." You wouldn't say "passing it the |foo| as arg."